static/monorail-2.0.yaml

```diff
@@ -298,7 +298,7 @@ definitions:
     type: object
 info:
   title: RackHD 2.0
-  version: 0.0.1
+  version: 0.1.0
```

Comment:
For such simple description modification, how about only change the last version number? 0.0.1 => 0.0.2?